### DISCUSSION TITLE
about Linux 中共享对象的写回

### DISCUSSION THREAD

**Discussion by PYGangGang** [liusy58/WhyThis#24] [General]
比如我们用 mmap 映射了一个磁盘上的文件为共享对象，并对它做了一些写操作。当内核决定牺牲掉其中的一些 dirty 的页面时，是将它立即写回到原文件还是暂时写到 swap space 呢？

**Comment by liusy58** [comment]
这个问题应该要去看源码了，稍后会回答这个问题。
